apps/web/src/pages/txs.tsx

```diff
@@ -34,6 +113,146 @@ const Txs: NextPage = function () {
   }, [rawTxsData]);
   const { transactions, totalTransactions } = txsData || {};
 
+  const transactionRows = useMemo(() => {
+    return transactions
+      ? transactions.map((t: Transaction) => {
+          const {
+            hash,
+            from,
+            to,
+            blobs,
+            rollup,
+            blockNumber,
+            blobGasBaseFee,
+            blobGasMaxFee,
+            block,
+            blockTimestamp,
+          } = t;
+
+          const getTransactionsTableRowExpandItem = (
+            blobs?: Pick<
+              DeserializedBlob,
+              | "versionedHash"
+              | "commitment"
+              | "size"
+              | "dataStorageReferences"
+              | "proof"
+            >[]
+          ) => {
+            const headers = [
+              {
+                cells: [
+                  {
+                    item: "Blob Versioned Hash",
+                  },
+                  {
+                    item: "Size",
+                  },
+                ],
+                className: "dark:border-border-dark/20",
+                sticky: true,
+              },
+            ];
+
+            const rows = blobs
+              ? blobs.map((b) => ({
+                  cells: [
+                    {
+                      item: b.versionedHash,
```

Comment:
Let's turn this into a link to the blob details page

Reply:
Done :+1: